src/SWP/Bundle/CoreBundle/Model/FacebookInstantArticlesFeed.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Superdesk Web Publisher Core Bundle.
+ *
+ * Copyright 2016 Sourcefabric z.ú. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2016 Sourcefabric z.ú
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\Bundle\CoreBundle\Model;
+
+use SWP\Bundle\FacebookInstantArticlesBundle\Model\PageInterface;
+use SWP\Component\Common\Model\TimestampableTrait;
+use SWP\Component\MultiTenancy\Model\TenantAwareInterface;
+use SWP\Component\MultiTenancy\Model\TenantAwareTrait;
+use SWP\Component\Storage\Model\PersistableInterface;
+
+class FacebookInstantArticlesFeed implements TenantAwareInterface, PersistableInterface, FacebookInstantArticlesFeedInterface
```

Comment:
this class should be originally placed in FacebookInstantArticles component and be inherited

PersistableInterface and TimestampableTrait should then be used in the parent class, here you would only use TenantAwareTrait and protected $contentBucket; - the rest should be placed in parent class